res/controllers/Behringer-Extension-scripts.js

```diff
@@ -1307,6 +1348,14 @@
                 new components.EffectUnit(effectUnitDefinition.unitNumbers, true),
                 componentStorage,
                 ["onFocusChange", "shift", "unshift"]);
+            var shiftType = effectUnitDefinition.sendShiftedFor;
+            if (typeof shiftType === "function") {
+                effectUnit.forEachComponent(function(component) {
+                    if (component instanceof shiftType) {
+                        component.sendShifted = true;
```

Comment:
whats the `shiftType === "function"` for?

Reply:
The property `sendShiftedFor` is a filter tor the type of a JS component , e.g. `c.Button`, `c.Pot` or `c.Component`. If the user provides a string, number, boolean or object instead, the following `instanceof` statement will cause a runtime error.

Reply:
Mhmm makes sense, but its not self explaining. Could you add a comment that explains this?

Reply:
Of course, done.